app/services/scenes/scene-item.ts

```diff
@@ -93,11 +93,11 @@ export class SceneItem extends SceneItemNode {
   }
 
   getScene(): Scene {
-    return this.scenesService.views.getScene(this.sceneId);
+    return this.scenesService.views.getScene(this.sceneId) as Scene;
```

Comment:
Shouldn't we just modify the return type to `Scene | undefined` if we're not handling the `undefined` case here?